fiftyone/core/fields.py

```diff
@@ -689,6 +688,35 @@ def __str__(self):
         )
 
 
+class Group(EmbeddedDocument):
+    """A named group membership.
+
+    Args:
+        id (None): the group ID
+        name (None): the group name
+    """
+
+    id = ObjectIdField(default=lambda: str(ObjectId()), db_field="_id")
+    name = StringField()
+
+    @property
+    def _id(self):
+        return ObjectId(self.id)
+
+    def element(self, name):
+        return self.__class__(id=self.id, name=name)
+
+
+class GroupField(EmbeddedDocumentField):
```

Comment:
Is this required? Could `EmbeddedDocumentField` just be used. `GroupField` will be an extra class that the App will have to hardcode for the sidebar, etc.

Reply:
k I can remove `GroupField` and just use `EmbeddedDocumentField`

Reply:
Done!